packages/analytics-browser/src/default-tracking.ts

```diff
@@ -13,32 +13,40 @@ import {
  * Returns false if autocapture === false or if autocapture[event],
  * otherwise returns true
  */
-const isTrackingEnabled = (autocapture: AutocaptureOptions | boolean | undefined, event: keyof AutocaptureOptions) => {
+const isTrackingEnabled = (
+  autocapture: AutocaptureOptions | boolean | undefined,
+  event: keyof AutocaptureOptions,
+  defaultValue = false,
+) => {
   if (typeof autocapture === 'boolean') {
     return autocapture;
   }
 
-  if (autocapture?.[event] === false) {
-    return false;
+  if (typeof autocapture === 'object') {
```

Comment:
Although it's not somethign we expect, the below can error if autocapture is set to null since null is an object